cflags.SH

```diff
@@ -379,6 +379,16 @@ esac
 ;;
 esac
 
+case "$gccversion" in
+*"1"[23]*)
```

Comment:
I wonder if `12|13` might make the logic a bit easier to notice what's going on?

Reply:
Ugh sorry, what was I thinking? This is a glob match, not a regexp match. Ignore me.

Reply:
I added a comment above the new section.